GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h

```diff
@@ -23,15 +23,22 @@
 #import <AppKit/AppKit.h>
 #endif
 
+NS_ASSUME_NONNULL_BEGIN
+
 @class GIDVerifiableAccountDetail;
 @class GIDVerifiedAccountDetailResult;
 
-NS_ASSUME_NONNULL_BEGIN
+#if TARGET_OS_IOS
+/// Represents a completion block that takes a `GIDVerifiedAccountDetailResult` on success or an
+/// error if the operation was unsuccessful.
+typedef void (^GIDVerifyCompletion)(GIDVerifiedAccountDetailResult *_Nullable verifiedResult,
+                                    NSError *_Nullable error);
+#endif // TARGET_OS_IOS
```

Comment:
The `#endif` can be at the bottom so that the entire header is only available under iOS. In other words, let's just have one `#if TARGET_OS_IOS` at the top of the header, and one `#endif` at the bottom.

Reply:
Done.